doc/release-process.md

```diff
@@ -1,38 +1,59 @@
 Release Process
 ====================
 
-Before every release candidate:
+## Branch updates
 
-* Update translations (ping wumpus on IRC) see [translation_process.md](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md#synchronising-translations).
-* Update manpages, see [gen-manpages.sh](https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/README.md#gen-manpagessh).
-* Update release candidate version in `configure.ac` (`CLIENT_VERSION_RC`).
+### Before every release candidate
 
-Before every minor and major release:
+- Update translations (ping wumpus on IRC) see [translation_process.md](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md#synchronising-translations).
+- Update manpages, see [gen-manpages.sh](https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/README.md#gen-manpagessh).
+- Update release candidate version in `configure.ac` (`CLIENT_VERSION_RC`).
 
-* Update [bips.md](bips.md) to account for changes since the last release.
-* Update version in `configure.ac` (don't forget to set `CLIENT_VERSION_RC` to `0`).
-* Write release notes (see below).
-* Update `src/chainparams.cpp` nMinimumChainWork with information from the getblockchaininfo rpc.
-* Update `src/chainparams.cpp` defaultAssumeValid with information from the getblockhash rpc.
+### Before every major and minor release
+
+- Update [bips.md](bips.md) to account for changes since the last release.
```

Comment:
Could you please keep the previous formatting of `*` to minimize the diff and make review easier?

Reply:
Done.